aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -177,6 +177,30 @@ services.Configure<JwtBearerOptions>(
     });
 ```
 
+The API's JWT handler raises events that enable control over the authentication process using `JwtBearerEvents`. To provide support for API authorization, `AddIdentityServerJwt` registers its own event handlers.
```

Comment:
I stole some of the wording here from the [API reference](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.jwtbearer.jwtbearerevents?view=aspnetcore-2.2).

Reply:
That's what your supposed to do, steal wording from any authoritative MS source.